tools/coverage_system_tests.rc

```diff
@@ -1,3 +1,12 @@
 [run]
-include =
-    */library.py
+omit =
+    */__init__.py
```

Comment:
What do we gain by leaving this out?
Are we covering in unit tests?

Reply:
All the other files are covered by nifake unit tests. Only `_library.py` is not. 

Because we chose not to limit converters and attribute classes to what the driver actually uses, you will end up with a lot not tested by the driver's system_tests.